http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/LeoMetricsMonitor.scala

```diff
@@ -108,18 +109,18 @@ class LeoMetricsMonitor[F[_]](config: LeoMetricsMonitorConfig,
   private[monitor] def countRuntimesByDbStatus(allRuntimes: List[RuntimeMetrics]): Map[RuntimeStatusMetric, Int] = {
     val allContainers = for {
       r <- allRuntimes
-      // Exclude welder to not double-count runtimes. Assume every runtime has a jupyter or rstudio
-      // container, but not both.
-      c <- r.containers if c != RuntimeContainerServiceType.WelderService
+      // Only care about Jupyter, RStudio, or Azure image types.
+      // Assume every runtime has exactly 1 of these.
+      imageTypes = Set(RuntimeImageType.Jupyter, RuntimeImageType.RStudio, RuntimeImageType.Azure)
```

Comment:
Nit, but i would maybe rename these to be more specific:
- JupyterGce
- RStudioGce
- JupyterAzure